### PR TITLE
LPD 43785 | Modify ObjectAdminApiHelper.postRandomObjectDefinition to receive named arguments

### PR DESCRIPTION
**Pull request by dannielraposo**
See the following Jira ticket: [LPD-43785](https://liferay.atlassian.net/browse/LPD-43785)

### CODE REVIEW

**Comment by dannielraposo** [comment]
ci:test:playwright

**Comment by ccorreagg** [comment]
The failures do not seem related to the changes

File: modules/test/playwright/tests/object-web/objectEntries.spec.ts [hunk 1]
```diff
@@ -766,27 +762,25 @@ test.describe('Manage object entries through View Object Entries', () => {
...
-			await apiHelpers.objectAdmin.postRandomObjectDefinition(
-				{code: 0},
-				undefined,
-				'default',
-				'company',
-				objectField
-			);
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFolderExternalReferenceCode: 'default',
+				scope: 'company',
```
Comment (cgmonte) [inline modules/test/playwright/tests/object-web/objectEntries.spec.ts:767]: No need to pass `'company'` here, as this is the default value.

File: modules/test/playwright/tests/object-web/objectEntries.spec.ts [hunk 2]
```diff
@@ -766,27 +762,25 @@ test.describe('Manage object entries through View Object Entries', () => {
...
```
Comment (cgmonte) [inline modules/test/playwright/tests/object-web/objectEntries.spec.ts:780]: No need to pass `'company'` here, as this is the default value.

**Comment by cgmonte** [comment]
Hey, I made two comments that are only observations, they do not block forwarding this. Thanks bringing named arguments back!

**Comment by cgmonte** [comment]
ci:forward

**Pull request closed without merging**